skops/cli/_update.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+import argparse
+import json
+import logging
+import zipfile
+from pathlib import Path
+
+from skops.cli._utils import get_log_level
+from skops.io import dump, load
+from skops.io._protocol import PROTOCOL
+
+
+def _update_file(
+    input_file: str | Path,
+    output_file: str | Path | None = None,
+    inplace: bool = False,
+    logger: logging.Logger = logging.getLogger(),
+) -> None:
+    """Function that is called by ``skops update`` entrypoint.
+
+    Loads a skops model from the input path, converts to the current skops format, and
+    saves to output file.
+
+    Parameters
+    ----------
+    input_file : str, or Path
+        Path of input skops model to load.
+
+    output_file : str, or Path, default=None
+        Path to save the updated skops model to.
+
+    inplace : bool, default=False
+        Whether to update and overwrite the input file in place.
+
+    logger : logging.Logger, default=logging.getLogger()
+        Logger to use for logging.
+    """
+    if inplace:
+        if output_file is None:
+            output_file = input_file
+        else:
+            raise ValueError(
```

Comment:
AFAICT, there is no test case for this, right? Could you please add one? Not sure why codecov didn't pick this up...

Reply:
This is already tested actually. I have changed the test function name to make it a bit clearer: https://github.com/skops-dev/skops/pull/343/commits/7846077e3cf9448d2eb2a1147f37bbe3776b72bb